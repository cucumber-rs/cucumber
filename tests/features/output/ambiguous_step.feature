Feature: ambiguous
  Scenario: ambiguous
    Given foo is 0 ambiguous
