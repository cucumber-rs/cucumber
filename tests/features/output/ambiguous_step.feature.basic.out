Feature: ambiguous
  Scenario: ambiguous
   âœ˜  Given foo is 0 ambiguous
      Step failed:
      Defined: tests/features/output/ambiguous_step.feature:3:5
      Step match is ambiguous: Possible matches:
      foo is (\d+) --> tests/output.rs:11:1
      foo is (\d+) ambiguous --> tests/output.rs:19:1
